pkg/workloads/cortex/lib/type/predictor.py

```diff
@@ -179,6 +179,17 @@ def _validate_required_fn_args(impl, fn_name, args):
         )
 
 
+def uses_neuron_savedmodel():
+    value = os.getenv("CORTEX_ACTIVE_NEURON")
```

Comment:
nit: this can probably be one line, i.e. `return os.getenv("CORTEX_ACTIVE_NEURON") != None`